eng/pipelines/libraries/build-job.yml

```diff
@@ -0,0 +1,122 @@
+parameters:
+  buildConfig: ''
+  osGroup: ''
+  osSubgroup: ''
+  archType: ''
+  framework: netcoreapp
+  isOfficialBuild: false
+  timeoutInMinutes: 150
```

Comment:
can we adjust the timeout per leg? Often we needed to change it only for arm/arm64.

Reply:
Yes, we can override it in platform-matrix.